### PR TITLE
[3.x] Update Stunnel Url

### PR DESCRIPTION
**Pull request by Nbot07A**
### Description of changes
* update stunnel url

### Tests
* passed test_createami.py::test_build_image

### References
* https://github.com/aws/aws-parallelcluster/pull/4668
* https://github.com/aws/aws-parallelcluster-cookbook/pull/1588
* https://github.com/aws/aws-parallelcluster/pull/4533
* https://github.com/aws/aws-parallelcluster/pull/4535

### Checklist
- [ ] Make sure you are pointing to **the right branch** and add a label in the PR title (i.e. **2.x** vs **3.x**)
- [ ] Check all commits' messages are clear, describing what and why vs how.
- [ ] Make sure **to have added unit tests or integration tests** to cover the new/modified code.
- [ ] Check if documentation is impacted by this change.

Please review the [guidelines for contributing](../CONTRIBUTING.md) and [Pull Request Instructions](https://github.com/aws/aws-parallelcluster/wiki/Git-Pull-Request-Instructions).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -305,7 +305,7 @@
 default['cluster']['efs_utils']['sha256'] = '69d0d8effca3b58ccaf4b814960ec1d16263807e508b908975c2627988c7eb6c'
 default['cluster']['efs_utils']['tarball_path'] = "#{node['cluster']['sources_dir']}/efs-utils-#{node['cluster']['efs_utils']['version']}.tar.gz"
 default['cluster']['stunnel']['version'] = '5.67'
-default['cluster']['stunnel']['url'] = "https://www.stunnel.org/downloads/stunnel-#{node['cluster']['stunnel']['version']}.tar.gz"
+default['cluster']['stunnel']['url'] = "#{node['cluster']['artifacts_s3_url']}/stunnel/stunnel-5.67.tar.gz"
```
Comment (hanwen-pcluste) [inline attributes/default.rb]: Please reuse `#{node['cluster']['stunnel']['version']}` for the version

Comment (Nbot07A) [reply attributes/default.rb]: Do you mean on line 307 as `default['cluster']['stunnel']['version'] = #{node['cluster']['stunnel']['version']}`

Comment (Nbot07A) [reply attributes/default.rb]: Or on line 308 as `default['cluster']['stunnel']['url'] = "#{node['cluster']['artifacts_s3_url']}/stunnel/stunnel-#{node['cluster']['stunnel']['version']}.tar.gz `

Comment (hanwen-pcluste) [reply attributes/default.rb]: Change line 308

Comment (Nbot07A) [reply attributes/default.rb]: Ok, I changed that here [reuse stunnel version variable](https://github.com/aws/aws-parallelcluster-cookbook/pull/1623/commits/1ed48da4b685ecfdffbb31c1e53ea2eb81839851)

**Review by hanwen-pcluste — APPROVED** [review]

**Pull request merged**